website/project/metadata/other-studies-using-youth-data.json

```diff
@@ -17,7 +17,7 @@
 		"required": false
 	  }, {
 		"help_text": "",
-		"block_type": "question-label",
+		"block_type": "paragraph",
```

Comment:
We should be getting rid of the section headings and just using question-labels instead of hacking with paragraphs.